tools/cldr-code/src/main/java/org/unicode/cldr/util/CodePointEscaper.java

```diff
@@ -17,26 +19,41 @@ public enum CodePointEscaper {
     LF(0xA, "line feed"),
     CR(0xD, "carriage return"),
     SP(0x20, "space", "ASCII space"),
-    NBSP(0xA0, "no-break space"),
-
-    NSP(0x2009, "narrow space", "thin space"),
-    NNBSP(0x202F, "narrow no-break space", "thin no-break space"),
-
-    ZWNJ(0x200C, "zero-width non-joiner"),
-    ZWJ(0x200D, "zero-width joiner"),
-
-    ZWSP(0x200B, "word non-joiner", "zero-width space"),
-    ZWNBSP(0x2060, "word joiner", "zero-width no-break space"),
-
-    ALM(0x061C, "Arabic letter mark"),
-    LRM(0x200E, "left-right mark"),
-    RLM(0x200F, "right-left mark"),
+    NSP(0x2009, "narrow/thin space", "Also known as ‘thin space’"),
+    NBSP(0xA0, "no-break space", "Same as space, but doesn’t line wrap."),
+
+    NNBSP(0x202F, "narrow/thin no-break space", "Same as narrow space, but doesn’t line wrap."),
```

Comment:
I think "narrow no-break space" is fine as the shortName since it expands the initialism; I'd move the "/thin" part into an appendix to the description: "; also ❛no-break thin space’."